default/scripting/species/SP_LEMBALALAM.focs.txt

```diff
@@ -0,0 +1,123 @@
+Species
+    name = "SP_LEMBALALAM"
+    description = "SP_LEMBALALAM_DESC"
+    gameplay_description = "SP_LEMBALALAM_GAMEPLAY_DESC"
+    Native
+
+    tags = [ "ORGANIC" "AI_TAG_NO_INDUSTRY" "AI_TAG_GOOD_RESEARCH" "AI_TAG_BAD_POPULATION" "AI_TAG_AVERAGE_SUPPLY" "AI_GOOD_STEALTH"]
```

Comment:
AI_GOOD_STEALTH -> AI_TAG_GOOD_STEALTH

AI_TAG_BAD_POPULATION is reserved for the standard case. Do not use it in this case.